app/javascript/components/pages/Patientinfo.test.js

```diff
@@ -2,16 +2,28 @@ import React from "react";
 import Enzyme, { shallow } from "enzyme";
 import Adapter from "enzyme-adapter-react-16";
 import PatientInfo from "./PatientInfo";
-import fetch from "node-fetch"
+
+const mockPatient = { first_name: "Stefani", last_name: "F" };
+
+global.fetch = () =>
+  Promise.resolve({ json: () => Promise.resolve(mockPatient) });
```

Comment:
Extra credit to move this mock into a global mock any test can use. you can use the setup file such that you don't need to rewrite this every time.

Reply:
will check this out for icebox